src/database/init.js

```diff
@@ -0,0 +1,9 @@
+import db from '../models';
+import env from '../config/env-config';
+
+if (env.NODE_ENV !== 'production') {
+  const syncOptions = env.NODE_ENV === 'test' ? { force: true } : {};
+  db.sequelize.sync(syncOptions).then(() => {
+    db.sequelize.close();
+  });
+}
```

Comment:
What does this file do?

Reply:
It's been removed. It was formally used to force sync the test DB upon migrations, but as discussed I have implemented a better solution for that and it's been removed.